Jenkinsfile

```diff
@@ -113,5 +113,33 @@ pipeline {
                 }
             }
         }
+        stage ('Build macOS') {
```

Comment:
For merging purposes this just verifies that building works and does not do any distribution